src/python/runpython.py

```diff
@@ -16,7 +16,10 @@
     sys.stdout.write("Windows based operating system detected\n")
     sys.stdout.flush()
     try:
-        os.system((sys.argv[1]+" "+sys.argv[2]+" "+sys.argv[3]))
+        try:
+            os.system((sys.argv[1]+" "+sys.argv[2]+" "+sys.argv[3]+" "+sys.argv[4]))
```

Comment:
Move command creation to line 13: `command = " ".join(sys.argv[1:])` to avoid unnecessary checks and duplicate code lines.
You can remove this try - except code and just use `os.system(command)` instead. And do the same for Linux & OSX branches as well.

Reply:
Done, thanks!